src/test/java/codeu/model/store/basic/ConversationStoreTest.java

```diff
@@ -59,6 +59,13 @@ public void testIsTitleTaken_false() {
     Assert.assertFalse(isTitleTaken);
   }
 
+  @Test
+  public void testGetLatestConversation() {
+    Conversation latestConversation = conversationStore.getLatestConversation();
+
+    Assert.assertEquals(CONVERSATION_ONE, latestConversation);
```

Comment:
I have no doubt that this is definitely getting the latest conversation after looking at your code, but I think that it might be best to add one or two more conversations to the mix when testing this function. Otherwise I don't think the test can show that we are getting the latest conversation added without a doubt, since there is only one conversation available.

Reply:
Since I'm accessing the list with (size-1) as index, I think adding one more object to each list is not necessary! Because having 2 objects against having only 1 wouldn't change the behavior at all 😄